### PR TITLE
636 muso cdi

### PR DESCRIPTION
**Pull request by gagbodamakou**
medic/config-muso#636

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -253,6 +253,16 @@ jobs:
...
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD_CHIS_NE }}
 
+    - name: Bundle cht_rci
+      uses: maierj/fastlane-action@v1.4.0
+      with:
+        lane: bundle
+        options: '{ "flavor": "cht_rci" }'
+      env:
+        ANDROID_KEYSTORE_PATH: cht_rci.keystore
+        ANDROID_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD_CHT_RCI }}
```
Comment (binokaryg) [inline .github/workflows/publish.yml:263]: Have these secrets been shared with the repo managers already?
Please refer to [this section](https://docs.communityhealthtoolkit.org/core/guides/android/branding/#:~:text=Use%20a%20safe%20channel%20to%20send%20the%20environment%20variables%20to%20the%20manager%20in%20charge) in the guide.

File: build.gradle [hunk 2]
```diff
@@ -360,6 +360,15 @@ android {
       dimension = 'brand'
       applicationId = 'org.medicmobile.webapp.mobile.chis_ne'
     }
+    cht_rci {
+      dimension = 'brand'
+      applicationId = 'org.medicmobile.webapp.mobile.cht_rci'
+    }
+    
+    cht_rci_test {
```
Comment (binokaryg) [inline build.gradle:368]: Do we need to build the test version as well?
We have the PR for this already as #240.
There I had [suggested](https://github.com/medic/cht-android/pull/240#discussion_r769852716) that we can create APK for test locally.

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -253,6 +253,16 @@ jobs:
         ANDROID_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD_CHIS_NE }}
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD_CHIS_NE }}
 
+    - name: Bundle cht_rci
```
Comment (binokaryg) [inline .github/workflows/publish.yml:256]: Unpack secrets, Assemble tasks missing?
Are you following [this](https://docs.communityhealthtoolkit.org/core/guides/android/branding/#:~:text=workflows/publish.yml.-,The%20Unpack%20secrets,-%E2%80%A6%20task%20unpacks%20and) guide?

**Review by binokaryg — COMMENTED** [review]
The logos are different for different sizes.
image

More comments inline.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (gagbodamakou) [reply .github/workflows/publish.yml:263]: @binokaryg ,  i haven't done it yet. Who is that manager and how to contact him?

File: build.gradle (hunk 2 above)
Comment (gagbodamakou) [reply build.gradle:368]: Ok, removed!

File: .github/workflows/publish.yml (hunk 3 above)
Comment (gagbodamakou) [reply .github/workflows/publish.yml:256]: Ok, Added!

**Comment by gagbodamakou** [comment]
> 
Hello @binokaryg , i'm done with the review and did the PUSH

**Comment by binokaryg** [comment]
> > 
> 
> Hello @binokaryg , i'm done with the review and did the PUSH

Did you check [this comment](https://github.com/medic/cht-android/pull/247#pullrequestreview-903847631)?
It looks like the icons are not updated for all sizes:
For example, these two look different:
- [src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png](https://github.com/medic/cht-android/blob/3abb5dc1830e488d80e10df666c67650c3931bf0/src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png)
- [src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png](https://github.com/medic/cht-android/blob/3abb5dc1830e488d80e10df666c67650c3931bf0/src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png)

File: .github/workflows/publish.yml (hunk 1 above)
Comment (gagbodamakou) [reply .github/workflows/publish.yml:281]: @binokaryg ? what about this?

Comment (derickl) [reply .github/workflows/publish.yml:281]: I can help you add those keys @gagbodamakou

**Comment by gagbodamakou** [comment]
> > > 
> > 
> > 
> > Hello @binokaryg , i'm done with the review and did the PUSH
> 
> It looks like the icons are not updated for all sizes: For example, these two look different:
> 
> * [src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png](https://github.com/medic/cht-android/blob/3abb5dc1830e488d80e10df666c67650c3931bf0/src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png)
> * [src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png](https://github.com/medic/cht-android/blob/3abb5dc1830e488d80e10df666c67650c3931bf0/src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png)

I think you are on the wrong commit.
here is where to check

**Comment by binokaryg** [comment]
> I think you are on the wrong commit.
here is where to check https://github.com/Muso-Health/cht-android/commit/3abb5dc1830e488d80e10df666c67650c3931bf0

I was not looking at a single commit but the whole PR.
Here are the links from your fork:
- https://github.com/Muso-Health/cht-android/blob/636-muso-cdi/src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png
- https://github.com/Muso-Health/cht-android/blob/636-muso-cdi/src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png

File: .github/workflows/publish.yml (hunk 1 above)
Comment (gagbodamakou) [reply .github/workflows/publish.yml:281]: Hi  @derickl, Thank you for your support.
Is it possible to have [1 min cal](https://meet.google.com/xxd-gnrn-twq?authuser=0)l to understand this process? it is a little bit unclear for me.
thks

**Comment by gagbodamakou** [comment]
> > I think you are on the wrong commit.
> > here is where to check [Muso-Health@3abb5dc](https://github.com/Muso-Health/cht-android/commit/3abb5dc1830e488d80e10df666c67650c3931bf0)
> 
> I was not looking at a single commit but the whole PR. Here are the links from your fork:
> 
> * https://github.com/Muso-Health/cht-android/blob/636-muso-cdi/src/cht_rci_test/res/mipmap-mdpi/ic_launcher.png
> * https://github.com/Muso-Health/cht-android/blob/636-muso-cdi/src/cht_rci_test/res/mipmap-hdpi/ic_launcher.png

Is it ok now?

**Comment by binokaryg** [comment]
> Is it ok now?
Those two still look different:
image

**Comment by gagbodamakou** [comment]
> 

Sorry, I didn't check the test Apk.
it is all ok now

File: .github/workflows/publish.yml (hunk 1 above)
Comment (derickl) [reply .github/workflows/publish.yml:281]: I can join on the link in the next 45 min. Is that OK @gagbodamakou ?

Comment (gagbodamakou) [reply .github/workflows/publish.yml:281]: I will be in a meeting at that  time, let say after 2pm GMT +1, if you are available

Comment (derickl) [reply .github/workflows/publish.yml:281]: That works

**Review by binokaryg — COMMENTED** [review]
Some changes requested.

**Comment by gagbodamakou** [comment]
Hello @binokaryg , 
We decided to change the icons to the ones used by Muso Mali, I also applied the other changes required
thks

File: .github/workflows/publish.yml (hunk 1 above)
Comment (gagbodamakou) [reply .github/workflows/publish.yml:280]: > Hi @gagbodamakou I got caught up with other engagements. Let me know when you would be available within the hour

Great, I am now free of all meetings

**Review by binokaryg — APPROVED** [review]
Thanks for addressing the feedback. Please merge after making sure the variables have been added to the repo secrets.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (derickl) [reply .github/workflows/publish.yml:280]: Seems we keep missing each other. I'm on that link for the next 30 min @gagbodamakou

**Pull request merged**